curriculum/challenges/english/25-front-end-development/quiz-computer-basics/66ed8fb9f45ce3ece4053eac.md

```diff
@@ -171,45 +171,45 @@ Windows File Explorer
 
 #### --text--
 
-What is not a good practice when naming files for later use?
+What is NOT a good practice when naming files for later use?
 
 #### --distractors--
 
-Putting an underscore or hyphen between words
+Putting an underscore or hyphen between words.
 
 ---
 
-Use a casing style such as camelCase and sticking with it
+Use a casing style such as camelCase and sticking with it.
 
 ---
 
-Putting a version number at the end of a document like `summaryReport_v2.docx`
+Putting a version number at the end of a document like `summaryReport_v2.docx`.
 
 #### --answer--
 
-Being lazy and pounding your keyboard like `asdfloweiueld` to get it over with
+Being lazy and pounding your keyboard like `asdfloweiueld` to get it over with.
```

Comment:
Can we come up with something plausible here? Maybe "using spaces" - not sure if that's mentioned though.

Reply:
Updated this one to
> Using non-descriptive names like `file1.txt`.